### ISSUE TITLE
Obtaining SE on the link scale for predictions of clmm models

### ISSUE THREAD

**Issue by frousseu** [rvlenth/emmeans#155]
Hello

I'm trying to obtain SE on the link scale from clmm models predictions. From what I understand, with `mode = "prob"`, SE are converted to the probability scale using the Delta method.

I tried obtaining SE on the link scale using `mode = "linear.predictor"`, but I'm unable to generate a reference grid containing the different levels of the response.

```r
library(ordinal)
library(emmeans)

m <- clmm(rating ~ temp + contact + (1|judge), data = wine)	
rg<-ref_grid(m,at=list(temp="warm",rating="1"))
rg

#  'emmGrid' object with variables:
#     temp = warm
#     contact = no, yes

e<-emmeans(m, ~ rating | temp, mode = "linear.predictor")

# Error in emmeans(m, ~rating | temp, mode = "linear.predictor") : 
#   No variable named rating in the reference grid

e<-emmeans(rg, ~ rating | temp, mode = "linear.predictor")

# Error in emmeans(m, ~rating | temp, mode = "linear.predictor") : 
#   No variable named rating in the reference grid
```
Is there a way to obtain SE on the link scale?
Thank you

**Comment by rvlenth** [comment]
First, note in your example, `rating` is not one of the variables in the reference grid; so obviously, `rating` cannot be a factor in determining EMMs. Second, the default is the latent scale, not the link scale (see the vignette for "models", section O: https://cran.r-project.org/web/packages/emmeans/vignettes/models.html#O). 

Estimates on the link scale are logits (or other specified link) of the *cumulative* probabilities at the cut points *between* ratings. To obtain these, use `mode = "linear.predictor"` in the `emmeans()` call. For your example, the variables in the reference grid will be `temp`, `contact`, and `cut` (see the third paragraph of the previously mentioned vignette). Again, this reference grid does not involve `rating`. The `rating` variable only appears with `mode = "prob"`, and it is obtained by first doing `mode = `cum.prob` (that is when the delta method comes in), and then differencing the results between `cut` levels to obtain the estimated probabilities for each rating. The latter step is essentially a contrast of the cumulative probabilities.
